USAGE.rst

```diff
@@ -41,51 +43,46 @@ Step-by-Step
    # clone the LWFA example to $HOME/picInputs/myLWFA
    pic-create $PICSRC/examples/LaserWakefield/ $HOME/picInputs/myLWFA
 
-Now edit ``$HOME/picInputs/myLWFA/include/picongpu/simulation_defines/param/*`` to change the :ref:`physical configuration of this input set <usage-params>`.
+   # switch to your input directory
+   cd $HOME/picInputs/myLWFA
 
-Now edit ``$HOME/picInputs/myLWFA/etc/picongpu/*.cfg`` to adjust :ref:`runtime parameters (simulation size, number of GPUs, plugins, ...) <usage-cfg>`.
+PIConGPU is controlled via two kinds of input sets: compile-time options and runtime options.
+
+Edit the :ref:`.param files<usage-params>` inside ``include/picongpu/simulation_defines/param/``.
```

Comment:
not relevant here, see above. users shall only interface with `.param` files, everything else is a hack for power-users.